cms/static/cms/js/modules/cms.plugins.js

```diff
@@ -604,6 +607,9 @@ $(document).ready(function () {
 			// cancel if quicksearch is focues
 			if(this.focused) return false;
 
+			//fix issue #4407
+			nav.parents('.cms_draggable').data( 'menustate', 0 ).trigger('cms.structureboard.menustate.change');
```

Comment:
closest would not work here, the menu overlays more than the direct draggable parent